sdk/package-lock.json

```diff
@@ -1,7093 +1,8 @@
 {
   "name": "@lightbend/akkaserverless-javascript-sdk",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
The lock file was updated to v2 for npm 7 (in #19). We probably don't want to keep going back and forth, so using node 14 with upgraded npm 7 will be required locally now.

Reply:
Ah, accidentally committed mostly

Reply:
still on `v1`